web/public/locales/fi.json

```diff
@@ -98,7 +98,7 @@
     "hideWindLayer": "Piilota tuulikerros",
     "showSolarLayer": "Näytä aurinkokerros",
     "hideSolarLayer": "Piilota aurinkokerros",
-    "toggleDarkMode": "Vaihda tummaa moodia",
+    "changeTheme": "Vaihda tummaa moodia",
```

Comment:
Maybe it would be better to remove this than renaming it? That way we fall back to the english translation and not a presumably false finnish translation (this is applicable to more languages as well).

Reply:
I'm going to make some generalizations here so they might not be 100% correct but I think it would be more clear to people if the text itself was accurate (i.e. deleting the existing one) instead of something similar (but not quite right) in their own language.

I also think people are more prone to add missing content then changing inaccurate content (just a personal stance though).

But I don't think we should consider this blocking for the PR as a whole.